core/src/OpenMPTarget/Kokkos_OpenMPTarget_Parallel.hpp

```diff
@@ -454,13 +455,16 @@ class ParallelReduce<FunctorType, Kokkos::RangePolicy<Traits...>, ReducerType,
   void execute() const {
     if constexpr (HasJoin) {
       // Enter this loop if the Functor has a init-join.
+      printf("Execute HasJoin in RangePolicy \n");
```

Comment:
I am not sure that you mean to leave this print statement in your code.

Reply:
Oh this was a mistake. I forgot to delete these after debugging 🤦 . Deleted them. Thanks @lucbv